jsonschema2md/__init__.py

```diff
@@ -316,9 +325,20 @@ def _construct_description_line(self, obj: dict[str, Any], add_type: bool = Fals
                 description_line.append(_("Cannot contain unevaluated properties."))
 
         if "$ref" in obj:
+            url = urlsplit(obj["$ref"])
+            if url.fragment and not url.path:
+                ref_link = f"#{quote(url.fragment[1:])}"
+            elif self.base and (url.netloc == self.base or url.path.startswith(self.base)):
```

Comment:
TODO: Maybe rename base to `domain` because that's what's being checked here. Or at the very least strip the scheme on definition.

Reply:
+1 to `domain` or `netloc` :-)